### PR TITLE
Changes to improve the package

### PR DESCRIPTION
**Pull request by eullercristian**
Remove bower, using only node_modules
Refactor node_modules versions
Add bootstrap, jquery, jquery-masked, lodash
Hability libraries to use of vue.js

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,10 +1,36 @@
...
+    "jquery-masked-input": "^0.1.2",
+    "jquery-maskmoney": "^3.0.2",
+    "laravel-elixir": "^5.0.0",
+    "laravel-elixir-vueify": "^1.0.2",
+    "lodash": "^4.13.1",
+    "vue": "^1.0.24",
+    "vue-alert": "^1.0.1",
+    "vue-hot-reload-api": "^1.3.2",
+    "vue-mask": "^1.0.0",
+    "vue-resource": "^0.7.0",
```
Comment (vinicius73) [inline package.json:33]: i prefer axios intead of vue-resource

File: gulpfile.js [hunk 2]
```diff
@@ -1,26 +1,46 @@
 var elixir = require('laravel-elixir');
```
Comment (vinicius73) [inline gulpfile.js:1]: I find it more productive to migrate directly to webpack/laravel-mix

Comment (eullercristian) [inline gulpfile.js]: @vinicius73  I agree with you, approve the PR in this moment. And i will refactor, i will work in others improves :)

Comment (eullercristian) [inline gulpfile.js]: @vinicius73  I have a question for you, webpack/laravel-mix its works in Laravel 5.1? So, this branch is refer to Laravel 5.1

**Pull request merged**